.travis.yml

```diff
@@ -1,12 +1,12 @@
-language: csharp
-dotnet: 2.1.301
-mono: none
-dist: trusty
+language: minimal
```

Comment:
It's fine like this, in the S3 image pull request I'll be running unit tests in travis, so I might revert this at some point, fyi.